tests/e2e/alert-overview-save.spec.js

```
@@ -0,0 +1,132 @@
+import { test, expect } from '@playwright/test'
+
+const CHART = 'mariadb-alerts'
+const FOLDER = 'deployments/e2e-save-test/dev'
+const FOLDER_BASENAME = 'dev'
+const ALERT_TYPE_LABEL = 'latency_slow_queries'
```

ESLint

[error] 'ALERT_TYPE_LABEL' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)